# awesome-genome-visualization

A list of interesting genome visualizers, genome browsers, or
genome-browser-like implementations

**See the new companion website here https://cmdcolin.github.io/awesome-genome-visualization/**

Follow the Twitter account here too :) https://twitter.com/awesomegenomev1

## Commercial
- [10x genomics](https://support.10xgenomics.com/genome-exome/software/visualization/latest/structural-variants)
- [Alamut](https://www.interactive-biosoftware.com/alamut-visual/)
- [ATCC Genome Portal (American Type Culture Collection)](https://docs.onecodex.com/en/articles/3996697-using-the-genome-browser)
- [Benchling](https://www.benchling.com/)
- [CLC Genomics workbench](https://digitalinsights.qiagen.com/news/blog/discovery/structural-variant-detection-using-clc-genomics-workbench/)
- [DNASTAR](https://www.dnastar.com/)
- [Geneious](https://www.geneious.com/)
- [Genestack](https://genestack.com/blog/2015/05/28/navigation-in-genestack-genome-browser/)
- [Golden Helix](http://goldenhelix.com/GenomeBrowse/)
- [Lucid viewer](https://lucidalign.com/)
- [MacVector](https://macvector.com/)
- [Persephone](https://persephonesoft.com/)
- [SnapGene](https://www.snapgene.com/)
- [Strand NGS](https://www.strand-ngs.com/)
- [Bionano](https://bionanogenomics.com/technology/structural-variation/)
- [Circa](https://omgenomics.com/circa)
## MSA
- [AA](http://www.bioinformatics.org/strap/aa) (also see STRAP)
- [abrowse](https://github.com/ihh/abrowse)
- [alignment.js](https://github.com/veg/alignment.js)
- [AlignmentComparator](http://bioinfweb.info/AlignmentComparator/) (Uses http://bioinfweb.info/LibrAlign/)
- [AlignmentViewer](https://alignmentviewer.org/)
- [AliView](http://www.ormbunkar.se/aliview/)
- [BioJS/MSA](https://msa.biojs.net/) [(pub)](https://academic.oup.com/bioinformatics/article/32/22/3501/2525598?login=true)
- [Boxshade](https://embnet.vital-it.ch/software/BOX_form.html)
- [ESPript](http://espript.ibcp.fr/ESPript/ESPript/)
- [Jalview](https://www.jalview.org/)
- [JSAV](http://www.bioinf.org.uk/software/jsav) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4304231/)
- [LogoJS](https://logojs.wenglab.org/app/gallery/)
- [MSABrowser](https://thekaplanlab.github.io/)
- [msaR](https://github.com/zachcp/msaR) (uses BioJS/MSA)
- [MView](https://www.ebi.ac.uk/Tools/msa/mview/)
- [NCBI MSA Viewer](https://www.ncbi.nlm.nih.gov/projects/msaviewer/)
- [pyBoxshade](https://github.com/mdbaron42/pyBoxshade) (also see boxshade)
- [pymsaplotter](https://github.com/orangeSi/pymsaploter)
- [React MSAViewer](https://github.com/plotly/react-msa-viewer)
- [react-msaview](https://github.com/GMOD/react-msaview)
- [seqotron](https://github.com/4ment/seqotron)
- [STRAP](http://www.bioinformatics.org/strap/) (also see AA)
- [TeXShade](http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/texshade/texshade.pdf)
- [Wasabi](http://wasabiapp.org/)
- [SNPViz](http://www.yeastrc.org/snipviz/4.HTML_Config_retrieve_newick_and_fasta_from_server/snip_viz_HTML_config_with_newick_clustering.html)
- [ALVIS (MSA viewer)](https://www.ebi.ac.uk/research/goldman/software/alvis) (Introduces sequence bundles concept, also see web app version https://www.ebi.ac.uk/goldman-srv/sequencebundles/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4856975/)
## CNV
- [aCNVViewer](https://github.com/FJD-CEPH/aCNViewer) [(pub)](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0189334)
- [CNVkit](https://cnvkit.readthedocs.io/en/stable/plots.html) [(pub)](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004873)
- [BAMScale](https://github.com/ncbi/BAMscale) [(pub)](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00343-x)
- [cnvCurator](http://www.acsu.buffalo.edu/~lm69/cnvCurator) [(pub)](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0766-y)
- [GenomeSpy](https://genomespy.app/)
- [covviz](https://brwnj.github.io/covviz/)
- [PURPLE](https://github.com/hartwigmedical/hmftools/blob/master/purity-ploidy-estimator/README.md) (intermutation-distance rainfall plots/katagesis plots are not technically in genomic coordinates, but do indicate genomic cluster of variatnts)
- [bctools cnv](https://samtools.github.io/bcftools/howtos/cnv-calling.html)
- [mocha](https://github.com/freeseek/mocha#plot-results) (Has a full analysis pipeline associated with end result visualizations)
- [genevisR](https://bioconductor.org/packages/release/bioc/vignettes/GenVisR/inst/doc/Intro.html)
## Static
- [BAMSnap](https://github.com/parklab/bamsnap)
- [ChIA-Pipe](https://github.com/TheJacksonLaboratory/ChIA-PIPE) [(pub)](https://www.biorxiv.org/content/10.1101/506683v1.full.pdf)
- [Chimeraviz](https://github.com/stianlagstad/chimeraviz)
- [CNView](https://github.com/RCollins13/CNView)
- [CNVPlot](https://github.com/dantaki/CNVplot/)
- [CNVpytor](https://github.com/abyzovlab/CNVpytor)
- [DNAPlotLib](https://github.com/VoigtLab/dnaplotlib)
- [ExonIntron (from Wormbase)](http://wormweb.org/exonintron)
- [FeatureViewer](https://github.com/calipho-sib/feature-viewer)
- [Geneviz](https://jrderuiter.github.io/geneviz/usage.html)
- [Genome STRiP](http://software.broadinstitute.org/software/genomestrip/)
- [GenomeTools](http://genometools.org/)
- [GenomeView](https://github.com/nspies/genomeview)
- [genoPlotR](http://genoplotr.r-forge.r-project.org/)
- [ggbio](http://tengfei.github.io/ggbio/) [(pub)](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-8-r77)
- [GGgenes](https://github.com/wilkox/gggenes)
- [GGsashimi](https://github.com/guigolab/ggsashimi)
- [gtrellis](http://bioconductor.org/packages/devel/bioc/vignettes/gtrellis/inst/doc/gtrellis.html)
- [Gviz](https://bioconductor.org/packages/release/bioc/html/Gviz.html)
- [Hagfish](https://github.com/mfiers/hagfish/wiki/Plots)
- [HiCPlotter](https://github.com/kcakdemir/HiCPlotter)
- [JVarKit/BamToSVG](http://lindenb.github.io/jvarkit/BamToSVG.html)
- [Lollipops](https://github.com/joiningdata/lollipops)
- [Mason](http://www.yeastrc.org/mason/)
- [Methplotlib](https://www.biorxiv.org/content/10.1101/826107v1.full.pdf)
- [Pairoscope](http://pairoscope.sourceforge.net/)
- [pauvre](https://github.com/conchoecia/pauvre) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6991124/figure/fig-3/)
- [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)
- [RACER](https://oliviasabik.github.io/RACERweb/articles/IntroToRACER.html)
- [RepViz](https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-019-4473-z)
- [Samplot](https://github.com/ryanlayer/samplot)
- [SashimiPlot](https://miso.readthedocs.io/en/fastmiso/sashimi.html)
- [shabam](https://github.com/dlrice/shabam)
- [SparK](https://github.com/harbourlab/SparK) [(pub)](https://www.biorxiv.org/content/10.1101/845529v1.full.pdf)
- [Spliceclust](https://github.com/pkimes/spliceclust)
- [Sushi](https://www.bioconductor.org/packages/release/bioc/vignettes/Sushi/inst/doc/Sushi.pdf)
- [svist4get](https://link.springer.com/article/10.1186/s12859-019-2706-8) [(pub)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2706-8)
- [svv](https://github.com/ryanlayer/svv)
- [svviz](https://github.com/svviz/svviz) [(pub)](https://pubmed.ncbi.nlm.nih.gov/26286809/)
- [svviz2](https://github.com/nspies/svviz2) [(pub)](https://pubmed.ncbi.nlm.nih.gov/26286809/)
- [trackViewer](https://bioconductor.org/packages/release/bioc/vignettes/trackViewer/inst/doc/trackViewer.html)
- [circlize](https://github.com/jokergoo/circlize)
- [mummer2circos](https://github.com/metagenlab/mummer2circos)
- [cgplot](https://github.com/dfguan/cgplot)
## Alignment viewer
- [GenomeView (java app)](https://genomeview.org/content/quick-start-guide)
- [Chromatic](https://chromatic.nci.nih.gov/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5987889/)
- [Gambit](http://labsergen.langebio.cinvestav.mx/bioinformatics/jacob/?p=473)
- [LookSeq](https://www.sanger.ac.uk/tool/lookseq/)
- [udon](https://github.com/ocxtal/udon) (Uses an advanced data structure for pileup, so visually a basic example but likely just a small demo)
## Comparative
- [GSSPlayground](https://github.com/orangeSi/GSSplayground)
- [Aequatus](https://github.com/TGAC/Aequatus)
- [AliTV](https://alitvteam.github.io/AliTV/d3/AliTV.html)
- [Artemis comparison tool (ACT)](https://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act)
- [Biodalliance comparative demo](http://biodalliance.org/dev/test-comparative.html)
- [Chromatiblock](https://github.com/mjsull/chromatiblock) [(pub)](https://www.biorxiv.org/content/10.1101/800920v2.full.pdf)
- [CMap](http://gmod.org/wiki/CMap)
- [CoGe](https://genomevolution.org/coge/)
- [Comparative assembly hub / "snake track"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296145/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296145/)
- [CrossBrowse](https://github.com/shenkers/CrossBrowse) [(pub)](https://www.biorxiv.org/content/10.1101/272880v1.full)
- [CVit](https://sourceforge.net/projects/cvit/)
- [Cvit.js](https://github.com/LegumeFederation/cvitjs)
- [EasyFig](https://mjsull.github.io/Easyfig/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3065679/)
- [GBrowse_syn](http://gmod.org/wiki/GBrowse_syn)
- [Genome Context Viewer](https://www.legumefederation.org/gcv/phytozome_10_2/)
- [Genome-plots-processing](https://github.com/filip-husnik/genome-plots-processing)
- [Genomicus](https://www.genomicus.biologie.ens.fr/genomicus/)
- [gggenomes](https://github.com/thackl/gggenomes)
- [GIVE](https://zhong-lab-ucsd.github.io/GIVE_homepage/)
- [mauve-viewer](https://github.com/nconrad/mauve-viewer)
- [MCScan (python version)](https://github.com/tanghaibao/jcvi/wiki/MCscan-(Python-version))
- [MCScanX](https://github.com/wyp1125/MCScanX)
- [mGSV (multiple genome synteny viewer)](https://github.com/kashmatic/mGSV)
- [MizBee](http://www.cs.utah.edu/~miriah/mizbee/Overview.html) [(pub)](https://ieeexplore.ieee.org/document/5290692)
- [Multiple genome viewer](http://www.informatics.jax.org/mgv/)
- [OMA local synteny browser](https://omabrowser.org/oma/synteny/ADH1A_HUMAN/)
- [pafr](https://github.com/dwinter/pafr)
- [Parasight](https://baileylab.brown.edu/parasight/galframe.html) (Examples at https://ratparalogy.gs.washington.edu/)
- [syri/plotsr](https://schneebergerlab.github.io/syri/plotsr.html)
- [progressiveMauve viewer](http://darlinglab.org/mauve/user-guide/viewer.html)
- [SyMap](http://www.agcol.arizona.edu/software/symap/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105427/)
- [Synima](https://github.com/rhysf/Synima) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5697234/)
- [Synteny browser](https://github.com/TheJacksonLaboratory/syntenybrowser)
- [SynTView](http://hub18.hosting.pasteur.fr/SynTView/documentation/) [(pub)](undefined)
- [SynVisio](https://synvisio.github.io/)
- [Tripal MapViewer](https://academic.oup.com/database/article/doi/10.1093/database/baz100/5612723)
- [VISTA browser](http://pipeline.lbl.gov/cgi-bin/gateway2)
- [XMatchView](https://www.bcgsc.ca/resources/software/xmatchview)
- [FastANI](https://github.com/ParBLiSS/FastANI)
- [clinker](https://github.com/gamcil/clinker) [(pub)](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab007/6103786)
- [sockeye](https://www.bcgsc.ca/resources/software/sockeye) [(pub)](https://genome.cshlp.org/content/14/5/956.full)
- [spinteny](https://github.com/skinner/spinteny)
## Single cell
- [Millefy](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7053140/)
## GWAS
- [PureScript genetics browser](https://github.com/chfi/purescript-genetics-browser)
- [Locuszoom](http://locuszoom.org/) (Also see locuszoom.js http://locuszoom.org/locuszoomjs.php)
## Dotplot
- [shinyChromosome](http://150.109.59.144:3838/shinyChromosome/) [(pub)](https://www.sciencedirect.com/science/article/pii/S1672022919301883)
- [D-GENIES](http://dgenies.toulouse.inra.fr/)
- [Delly-maze](https://github.com/dellytools/maze)
- [Discoplot](https://github.com/mjsull/DiscoPlot)
- [Dot](https://dot.sandbox.bio/)
- [Dotlet](https://dotlet.vital-it.ch/)
- [Dotplot (chirimoyo)](https://chirimoyo.ac.uma.es/bitlab/portfolio/dotplot/)
- [iLambda/Dotplot](https://github.com/iLambda/dotplot)
- [dotPlotly](https://github.com/tpoorten/dotPlotly/)
- [Dottup](https://www.bioinformatics.nl/cgi-bin/emboss/help/dottup) (also seen here http://eichlerlab.gs.washington.edu/publications/chm1-structural-variation/data/GRCh37/heterochromatic_extensions.pdf)
- [FlexiDot](https://github.com/molbio-dresden/flexidot)
- [Gepard](http://cube.univie.ac.at/gepard) [(pub)](https://academic.oup.com/bioinformatics/article/23/8/1026/198110)
- [Jdot](https://github.com/LyonsLab/jdot)
- [last-dotplot](http://last.cbrc.jp/)
- [mcutils](https://github.com/mchaisso/mcutils) (https://twitter.com/mjpchaisson/status/1040363992206569472)
- [Minidot](https://github.com/thackl/minidot)
- [Mummer-idotplot](https://github.com/ryought/mummer-idotplot)
- [Mummerplot](http://mummer.sourceforge.net/)
- [ggplot2 mummerplot](https://jmonlong.github.io/Hippocamplus/2017/09/19/mummerplots-with-ggplot2/)
- [Redottable](https://github.com/s-andrews/redotable)
- [syntenyPlotByR](https://github.com/shingocat/syntenyPlotByR)
- [Dotter](https://sonnhammer.sbc.su.se/Dotter.html) [(pub)](https://pubmed.ncbi.nlm.nih.gov/8566757/)
- [JGI/IMG](https://img.jgi.doe.gov/) (Dotplot use MUMmer)
- [MashMap](https://github.com/marbl/MashMap/blob/master/scripts/generateDotPlot)
## Uncategorized
- [TnT](https://tnt.marlin.pub/articles/introduction)
- [Edinburgh-Genome-Foundry/DnaFeaturesViewer](https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer) [(pub)](https://academic.oup.com/bioinformatics/article-abstract/36/15/4350/5868559?redirectedFrom=PDF)
- [pyBamView](https://mgymrek.github.io/pybamview/)
- [bcftools roh](https://samtools.github.io/bcftools/howtos/roh-calling.html) (Plots runs of homozygosity, has interactive plot-roh.py helper too)
- [NanoMethViz](http://www.bioconductor.org/packages/release/bioc/html/NanoMethViz.html) [(pub)](https://www.biorxiv.org/content/10.1101/2021.01.18.426757v3)
- [Nextclade](https://clades.nextstrain.org/) (Related usage on nextstrain app also)
## Graph
- [AGB assembly graph browser](https://almiheenko.github.io/AGB/index.html) [(pub)](https://academic.oup.com/bioinformatics/article/35/18/3476/5306331)
- [Bandage](https://github.com/rrwick/Bandage/) [(pub)](https://academic.oup.com/bioinformatics/article/31/20/3350/196114)
- [gfaestus](https://github.com/chfi/gfaestus)
- [gGnomes.js](https://github.com/mskilab/gGnome.js)
- [graphgenomeviewer](https://cmdcolin.github.io/graphgenomeviewer/)
- [MoMi-G](https://github.com/MoMI-G/MoMI-G/)
- [odgi draw + odgi viz](https://pangenome.github.io/odgi/odgi_docs.html#_odgi_viz1) (Example from https://github.com/pangenome/pggb)
- [sequence tube map](https://github.com/vgteam/sequenceTubeMap)
- [Shasta](https://chanzuckerberg.github.io/shasta/ComputationalMethods.html#ReadGraph) (Uses graphviz)
- [LINX](https://github.com/hartwigmedical/hmftools/blob/master/sv-linx/README_VIS.md) (part of the PURPLE/GRIDSS/LINX pipeline)
- [GfaViz](undefined) [(pub)](https://academic.oup.com/bioinformatics/article/35/16/2853/5267826)
## Text based
- [Alan](https://github.com/mpdunne/alan)
- [Alv](https://github.com/arvestad/alv)
- [ASCIIGenome](https://github.com/dariober/ASCIIGenome)
- [Bamcov](https://github.com/fbreitwieser/bamcov)
- [bioSyntax](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2315-y)
- [BLAST+ (reading -outfmt 0 pairwise output is the original CLI bioinformatics!)](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
- [Hapviz](https://github.com/ekg/hapviz)
- [plotReads](http://campuspress.yale.edu/knightlab/ruddle/plotreads/)
- [Rna Alignment Viewers](http://biowiki.org/wiki/index.php?title=Rna_Alignment_Viewers&redirect=no) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109877/)
- [Sam2pairwise](https://github.com/mlafave/sam2pairwise)
- [Samtools depth visualization](https://twitter.com/yokofakun/status/1178686978541441025)
- [Samtools tview](http://www.htslib.org/)
- [SvABA](https://github.com/walaj/svaba#alignmentstxtgz)
- [VizAln (from HipSTR)](https://github.com/tfwillems/HipSTR)
## Circular
- [Anvio](https://peerj.com/articles/1319/)
- [BioCircos.js](http://bioinfo.ibp.ac.cn/biocircos/)
- [Circleator](http://jonathancrabtree.github.io/Circleator/index.html)
- [Circos](http://circos.ca/)
- [CircosJS](https://github.com/nicgirault/circosJS)
- [DNAPlotter](https://www.sanger.ac.uk/science/tools/dnaplotter)
- [GenomeProjector](https://github.com/gaou/g-language/wiki) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2636772/)
- [JupiterPlot](https://github.com/JustinChu/JupiterPlot)
- [OGDRAW](https://chlorobox.mpimp-golm.mpg.de/OGDraw.html) [(pub)](https://www.biorxiv.org/content/10.1101/545509v1.full.pdf)
- [Gview](https://server.gview.ca/examples)
## Synthetic biology
- [seqviz](https://github.com/Lattice-Automation/seqviz#viewer)
- [ApE (a plasmid editor)](https://jorgensen.biology.utah.edu/wayned/ape/)
- [Chopchop](http://chopchop.cbu.uib.no/results/1611256682447.8318/details/9)
## Special-purpose
- [Apollo](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC151184/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC151184/)
- [D3GB](http://d3gb.usal.es/)
- [DNASkittle](https://www.dnaskittle.com/)
- [EaSeq](https://easeq.net/screenshots/) [(pub)](https://www.nature.com/articles/nsmb.3180)
- [FluentDNA](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203487/)
- [Hawkeye](undefined) [(pub)](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2007-8-3-r34)
- [IslandPlot](https://bl.ocks.org/lairdm/c6c235dbfa6e6ee61565) (D3, SVG)
- [MagicViewer](http://bioinformatics.zj.cn/magicviewer/) [(pub)](https://academic.oup.com/nar/article/38/suppl_2/W732/1096410)
- [Panoptes](https://www.malariagen.net/apps/ag1000g/phase1-AR3/index.html)
- [MapView](https://sites.google.com/site/wjwdavy/) (links in original paper are dead, but URL provided here works) [(pub)](https://academic.oup.com/bioinformatics/article/25/12/1554/193807)
- [NGB](https://github.com/epam/NGB)
- [SeqCover](https://brentp.github.io/seqcover/#gene=AIFM1)
- [SEQing](https://github.com/malewins/SEQing)
- [SFARI](https://gene.sfari.org/database/human-gene/)
- [Short read assembly browser](https://rapdb.dna.affrc.go.jp/AssemblyBrowser2/index.html?ref=%22chr01%22&run_id=%22G4toI1%22&start=%22234780%22&end=%22236631%22)
- [SNPitty](https://bitbucket.org/ccbc/snpitty/src/master/) [(pub)](https://www.jmdjournal.org/article/S1525-1578(17)30166-6/fulltext)
- [Staden](https://sourceforge.net/projects/staden/)
- [TASUKE](https://ricegenome.dna.affrc.go.jp/)
- [TE-nest](https://pubmed.ncbi.nlm.nih.gov/18032588) (download link unknown)
- [Transposcope](https://academic.oup.com/bioinformatics/article-abstract/36/12/3877/5820971?redirectedFrom=fulltext)
- [UGENE](http://ugene.net/)
- [Varsome](https://varsome.com/security-validation/?next=/variant/hg19/NM_000088.3(COL1A1):c.658C%3ET)
- [Vials](http://vials.io/)
- [wasm bigwig demo browser](https://shk656461.github.io/index.html)
## Gene structure
- [Apollo aka WebApollo](https://genomearchitect.readthedocs.io/en/latest/)
## SV
- [Ribbon](http://genomeribbon.com/) [(pub)](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btaa680/5885081?redirectedFrom=fulltext)
- [Gremlin](http://compbio.cs.brown.edu/projects/gremlin/)
- [InGAP-SV](http://ingap.sourceforge.net/)
- [Introgression browser](https://git.wageningenur.nl/aflit001/ibrowser) [(pub)](https://onlinelibrary.wiley.com/doi/pdf/10.1111/tpj.12800)
- [SplitThreader](http://splitthreader.com/)
- [SMRT View](http://files.pacb.com/software/smrtanalysis/2.3.0/doc/smrtview/help/Webhelp/App_View_Epipro.htm)
- [asgart](https://github.com/delehef/asgart) [(pub)](https://academic.oup.com/bioinformatics/article/34/16/2708/4948616)
## Linear
- [Genoverse](https://github.com/wtsi-web/Genoverse)
- [Gnomad browser](https://github.com/macarthur-lab/gnomadjs)
- [GenomePaint](https://genomepaint.stjude.cloud/) (See also gallery here https://genomepaint.stjude.cloud/#figures) [(pub)](https://www.cell.com/cancer-cell/fulltext/S1535-6108(20)30659-0)
- [LinearDisplay.pl](https://github.com/JCVenterInstitute/LinearDisplay)
## Expression
- [GTEX Locus browser](https://gtexportal.org/home/locusBrowserPage/ACTN3)
## Population
- [GWAS catalog browser](https://www.ebi.ac.uk/gwas/variants/rs1558902)
- [Human genome dating](https://human.genome.dating/region/chr1_13250000) (made with vega/d3)
- [PopSV](https://github.com/jmonlong/PopSV/blob/master/3-Visualization.md)
## Assembly QC
- [Icarus](http://quast.sourceforge.net/icarus.html)
## General
- [BasePlayer](https://baseplayer.fi/) [(pub)](https://www.biorxiv.org/content/10.1101/126482v1.full)
- [Celera genome browser](https://www.csee.umbc.edu/~turner/presentations/bvw2002/sld009.htm)
- [IGB](https://bioviz.org/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4937187/)
- [IGV](https://igv.org/)
- [Savant](http://bioinformatics-ca.github.io/savant_genome_browser_lab_2015/) [(pub)](https://pubmed.ncbi.nlm.nih.gov/22638571/)
- [Tablet](https://ics.hutton.ac.uk/tablet/)
- [Biodalliance](http://www.biodalliance.org/)
- [Ensembl genome browser](https://useast.ensembl.org/Homo_sapiens/Location/View?r=17:63992802-64038237)
- [Ensembl genome browser 2020 edition](http://2020.ensembl.org/)
- [GBrowse+GBrowse 2](http://gmod.org/wiki/GBrowse)
- [GenomeMaps](http://www.genomemaps.org/) [(pub)](https://pubmed.ncbi.nlm.nih.gov/23748955/)
- [Gosling](https://gosling.js.org/)
- [HiGlass](https://higlass.io)
- [IGV.js](https://github.com/igvteam/igv.js/)
- [JBrowse](http://jbrowse.org/jbrowse1.html) (See also JBrowse plugin registry https://gmod.github.io/jbrowse-registry. Runs on the web or as a desktop app using Electron)
- [JBrowse 2](http://jbrowse.org/jb2) (See gallery for more examples https://jbrowse.org/jb2/gallery)
- [Kero-BROWSE](https://kero.hgc.jp/examples/CLCL/hg38/index.html)
- [NCBI Genome Data Viewer](https://www.ncbi.nlm.nih.gov/genome/gdv/)
- [Nucleome browser](https://vis.nucleome.org/v1/main.html)
- [Pileup.js](https://github.com/hammerlab/pileup.js/)
- [Trackster](https://galaxyproject.org/learn/visualization/)
- [UCSC genome browser](https://genome.ucsc.edu/)
- [Valis browser](https://valis.bio/)
- [WashU epigenomics browser](https://epigenomegateway.wustl.edu/)
- [Zenbu](http://fantom.gsc.riken.jp/zenbu/)
## Exotic
- [BigTop](https://blog.dnanexus.com/2019-05-21-bigtop-data-visualization/)
- [Cylindrical alignment app](https://sourceforge.net/projects/cylindrical-alignment-app/)
## Data grid
- [BrowseVCF](https://github.com/BSGOxford/BrowseVCF) [(pub)](https://pubmed.ncbi.nlm.nih.gov/27373737/)
- [CuteVCF](https://github.com/labsquare/CuteVCF)
## Epigenomics
- [Chip Monk](http://www.bioinformatics.babraham.ac.uk/projects/chipmonk/)
- [Chipster](https://chipster.csc.fi/) (Copy number tutorial https://chipster.csc.fi/manual/cn-tutorial.pdf)
- [cisGenome Browser](https://jhui2014.github.io/browser/screenshots.html)
- [Epilogos](https://epilogos.altius.org/)
- [Epiviz](https://epiviz.github.io/) [(pub)](https://www.biorxiv.org/content/10.1101/865295v1.full.pdf)
- [HiC browser](http://promoter.bx.psu.edu/hi-c/view.php)
- [HiPiler](http://hipiler.higlass.io)
- [Juicebox](https://github.com/aidenlab/Juicebox) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5596920/)
- [Juicebox.js](http://www.igv.org/doc/juiceboxjs.html) [(pub)](https://www.cell.com/fulltext/S2405-4712(18)30001-2)
- [Peax](https://github.com/Novartis/peax)
## Ideogram
- [chromoMap](https://www.biorxiv.org/content/10.1101/605600v1.full.pdf)
- [eweitz/Ideogram](https://github.com/eweitz/ideogram)
- [Ideogram](https://github.com/RCollins13/HumanIdiogramLibrary)
- [Ideoplot](https://github.com/mchaisso/Ideoplot)
- [karyoploteR](https://github.com/bernatgel/karyoploteR)
- [KaryotypeSVG](https://github.com/andreasprlic/karyotypeSVG)
- [NCBI Genome Decoration](https://www.ncbi.nlm.nih.gov/genome/tools/gdp/)
- [UCSC Genome Graphs](http://genome.ucsc.edu/cgi-bin/hgGenome)
## Mobile
- [iGenomics](https://www.biorxiv.org/content/10.1101/2020.02.11.944132v1)
## Pangenome
- [Pan-Tetris](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4547177/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4547177/)
- [Panacaea](https://github.com/JCVenterInstitute/PanACEA)
- [Panache](https://github.com/SouthGreenPlatform/panache)
- [panX](http://pangenome.tuebingen.mpg.de/) [(pub)](https://academic.oup.com/nar/article/46/1/e5/4564799)
- [RPAN (3kricedb)](http://cgm.sjtu.edu.cn/3kricedb/visualization/?tracks=DNA%2Cgene%2CPF)
## Protein
- [Protael](http://sanshu.github.io/protaelweb/) [(pub)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5963358/)
- [ProteinPaint](https://pecan.stjude.cloud/proteinpaint) [(pub)](https://www.nature.com/articles/ng.3466)
- [pViz](https://github.com/Genentech/pviz)
## Heatmap
- [UCSC Xena](https://xena.ucsc.edu/)
- [VIVA](https://www.biorxiv.org/content/10.1101/589879v1.full.pdf)
## Repeats
- [REPAVER](https://gitlab.com/gringer/bioinfscripts/) [(pub)](https://www.researchgate.net/publication/346507956_REPAVER_Paving_the_Way_for_Repeat_Discovery_in_Complex_Genomes?channel=doi&linkId=5fc58ba4a6fdcce95268fe60&showFulltext=true)
## Historical
- [Internet Contig Explorer](https://www.bcgsc.ca/resources/software/ice) [(pub)](https://genome.cshlp.org/content/13/6a/1244.long)
## QC
- [ALVIS (chimeric alignment viewer)](https://github.com/SR-Martin/alvis) [(pub)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04056-0)
- [vcfR](https://knausb.github.io/vcfR_documentation/) (image shows chromoqc output)
# Other lists

- [Awesome Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
- [Genocat](http://genocat.tools/) large collection of genomic visualizations with great review paper accompaniment https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.13727
- [awesome-biological-visualizations](https://github.com/keller-mark/awesome-biological-visualizations)

Send in PRs for more stuff!

## License

This README.md and TOOLS.json are CC0
https://creativecommons.org/publicdomain/zero/1.0/

## Remember

Always remember, "YOU ARE AWESOME"!!! (read in Ben Busby's voice, thanks Ben for reminding us of this always :)

